satellite/console/consoleweb/server.go

```diff
@@ -95,6 +95,7 @@ type Config struct {
 	NewProjectDashboard             bool    `help:"indicates if new project dashboard should be used" default:"false"`
 	NewNavigation                   bool    `help:"indicates if new navigation structure should be rendered" default:"true"`
 	NewObjectsFlow                  bool    `help:"indicates if new objects flow should be used" default:"true"`
+	NewAccessGrantFlow              bool    `help:"indicates if new access grant flow should be used" default:"true"`
```

Comment:
`default` should be `false` before merge.

Reply:
addressed in latest commit